README.md

```diff
@@ -121,177 +135,192 @@ Other custom configurations are referenced in this repository.
 
 3. Provide the operator permissions for pods (substitute your project for "my-project"):
 
-```bash
-oc adm policy add-scc-to-group redis-enterprise-scc system:serviceaccounts:my-project
-```
+    ```bash
+    oc adm policy add-scc-to-group redis-enterprise-scc system:serviceaccounts:my-project
+    ```
 
 4. Deploy the OpenShift operator bundle:
+    > NOTE: Update the `storageClassName` setting in `openshift.bundle.yaml` (by default its set to `gp2`).
 
-```bash
-oc apply -f openshift.bundle.yaml
-```
+    ```bash
+    oc apply -f openshift.bundle.yaml
+    ``` 
 
-Apply the `RedisEnterpriseCluster` resource with RHEL7 based images
+5. Redis Enterprise Cluster custom resource - `RedisEnterpriseCluster`
 
-```bash
-kubectl apply -f openshift/redis-enterprise-cluster_rhel.yaml
-```
+    Apply the `RedisEnterpriseCluster` resource with RHEL7 based images:
 
-### Configuration:
-The operator deploys with default configurations values, but those can be customized:
+    ```bash
+    kubectl apply -f openshift/redis-enterprise-cluster_rhel.yaml
+    ```
 
-Redis Image
-```yaml
-  redisEnterpriseImageSpec:
-    imagePullPolicy:  IfNotPresent
-    repository:       redislabs/redis
-    versionTag:       5.4.14-31
-```
+6. Redis Enterprise Database custom resource - `RedisEnterpriseDatabase` (Alpha feature)
 
-Persistence
-```yaml
-  persistentSpec:
-    enabled: true
-    volumeSize: "10Gi" # if you don't provide default is 5 times RAM size
-    storageClassName: "standard" #on AWS common storage class is gp2
-```
+   Create a `RedisEnterpriseDatabase` (REDB) by using Custom Resource.
+   The Redis Enterprise Operator can be instructed to manage databases on the Redis Enterprise Cluster using the REDB custom resource.
+    Example:
+    ```yaml
+    cat << EOF > /tmp/redis-enterprise-database.yml
+    apiVersion: app.redislabs.com/v1alpha1
+    kind: RedisEnterpriseDatabase
+    metadata:
+      name: redis-enterprise-database
+    spec:
+      redisEnterpriseCluster:
+        name: redis-enterprise
+      memorySize: 100MB
+    EOF
+    kubectl apply -f /tmp/redis-enterprise-database.yml
+    ```
+    Replace the name of the cluster with the one used on the current namespace.
+    All REDB configuration options are documented [here](redis_enterprise_database_api.md).
 
-Redis Enterprise Nodes (pods)
-```yaml
-  redisEnterpriseNodeResources:
-    limits:
-      cpu: "4000m"
-      memory: 4Gi
-    requests:
-      cpu: "4000m"
-      memory: 4Gi
-```
 
-User Name to be used for accessing the cluster. Default is demo@redislabs.com
-```yaml
-username: "admin@acme.com"
-```
+   > Optional: REDB admission controller (Alpha feature)
+   >
+   > When using the REDB Custom Resource Definition (Redis Enterprise Database) it is recommended to set up admission controller to improve input validation and catch configuration errors before they reach the cluster. The procedure is documented [here](admission/README.md).
```

Comment:
We cannot recommend using it and call it an Alpha feature. Let's decide which. Alpha or recommended. DB Controller is not an "Alpha" feature. We agreed we'll call the API alpha.